Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -459,6 +459,9 @@
 steps['RunDisplacedJet2018C']={'INPUT':InputInfo(dataSet='/DisplacedJet/Run2018C-v1/RAW',label='2018C',events=100000,location='STD', ls=Run2018C)}
 steps['RunCharmonium2018C']={'INPUT':InputInfo(dataSet='/Charmonium/Run2018C-v1/RAW',label='2018C',events=100000,location='STD', ls=Run2018C)}
 
+Run2018Cpilot={319348: [[1, 100]]}
+steps['RunJetHT2018C_nanoULremini']={'INPUT':InputInfo(dataSet='/JetHT/Run2018C-UL2018_MiniAODv2_pilot1-v1/MINIAOD',label='2018C',events=100000,location='STD', ls=Run2018Cpilot)}
```

Comment:
Why do we need to use pilot dataset here? Pilot dataset should be removed anyway when processing is done. Please move to real Legacy dataset, and contact PdmV to lock specific block at FNAL or CERN where relvals will run.

Reply:
FYI @chayanit

Reply:
Hi, 

Here is the realy Legay dataset which is still under production but can be used:
/JetHT/Run2018C-UL2018_MiniAODv2-v1/MINIAOD

https://cmsweb.cern.ch/das/request?view=list&limit=50&instance=prod%2Fglobal&input=dataset+dataset%3D%2FJetHT%2FRun2018C-UL2018_MiniAODv2*%2FMINIAOD+status%3DPRODUCTION